web/src/Web.App/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -283,4 +282,40 @@ private static AuthenticationBuilder AddDfeSignInOpenIdConnect(this Authenticati
                 };
             });
     }
+
+    private static async Task<(string[] schools, string[] trusts)> HandleTrustSchoolsAsync(Organisation organisation,
```

Comment:
I understand that it might be a bit tricky, but it would be amazing if these two methods could be abstracted away and unit tested, just for some additional assurance.